packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -145,87 +131,181 @@ export default {
+    md: (
         title: heading-m,
         text: body-l,
     ),
-    fontSizeLarge: (
+    lg: (
         title: heading-l,
         text: subheading-s,
     ),
-    fontSizeXLarge: (
+    xl: (
         title: heading-xl,
         text: subheading-s,
     ),
-    fontSizeXXLarge: (
+    xxl: (
         title: heading-xxl,
         text: subheading-l,
     )
 );
 
+$flexMargin: spacing(x3);
+
 .c-mediaElement {
     display: flex;
     width: 100%;
+}
+
+.c-mediaElement--col--reverse {
+    flex-direction: column-reverse;
 
-    &.c-mediaElement--reverse {
-        flex-direction: row-reverse;
+    & .c-mediaElement-content {
+        margin-top: $flexMargin;
     }
 }
+.c-mediaElement--col {
+    flex-direction: column;
 
-/**
- * Modifier – .c-mediaElement--stackOnNarrow
- *
- * Applies flex direction column on <=narrow
- */
-@include media('<=narrow') {
-    .c-mediaElement--stackWhenNarrow {
+    & .c-mediaElement-content {
+        margin-bottom: $flexMargin;
+    }
+}
+.c-mediaElement--row {
+    flex-direction: row;
+}
+.c-mediaElement--row--reverse {
+    flex-direction: row-reverse;
+}
+
+
+@each $name, $value in $breakpoints {
+
+    /**
+    * Modifier – .c-mediaElement--gte--$name--col
+    * Modifier – .c-mediaElement--gte--$name--col--reverse
+    * Modifier – .c-mediaElement--gte--$name--row
+    * Modifier – .c-mediaElement--gte--$name--row--reverse
+     *
+     * Applies flex direction column on $name
+     */
+    @include media('>='+$name) {
+        .c-mediaElement-col--gte--#{$name} {
             flex-direction: column;
-        /**
-         * Modifier – .c-mediaElement--reverse
-         *
-         * When stacked in flex col applies col-reverse
-         */
-        &.c-mediaElement--reverse {
-            flex-direction: column-reverse;
 
-            & .c-mediaElement-text {
-                margin-bottom: spacing(x3);
+            & .c-mediaElement-content {
+                margin-bottom: $flexMargin;
             }
         }
+        .c-mediaElement-col--gte--#{$name}--reverse {
+            flex-direction: column-reverse;
 
-        & .c-mediaElement-title {
-            margin-top: spacing(x3);
+            & .c-mediaElement-content {
+                margin-top: $flexMargin;
+            }
+        }
+        .c-mediaElement-row--gte--#{$name} {
+            flex-direction: row;
+        }
+        .c-mediaElement-row--gte--#{$name}--reverse {
+            flex-direction: row-reverse;
         }
     }
-}
 
-/**
- * Modifier – .c-mediaElement--stack
- *
- * Applies flex direction column
- */
-.c-mediaElement--stack {
-    flex-direction: column;
+    /**
+    * Modifier – .c-mediaElement--gt--$name--col
+    * Modifier – .c-mediaElement--gt--$name--col--reverse
+    * Modifier – .c-mediaElement--gt--$name--row
+    * Modifier – .c-mediaElement--gt--$name--row--reverse
+    *
+    * Applies flex direction column on $name
+    */
+    @include media('>'+$name) {
+        .c-mediaElement--gt-#{$name}--col {
```

Comment:
Missing hyphen, I think?
```suggestion
        .c-mediaElement--gt--#{$name}--col {
```

Reply:
yea thanks great catch.